### DISCUSSION TITLE
What is period and range?

### DISCUSSION THREAD

**Discussion by shankarcse2010** [Mathieu2301/TradingView-API#6] [Q&A] [answered]
what is period and range in code? and where can i find in trading view?
can you help me?

market.on('logged', () => {
  market.initChart({
    symbol: 'custom_symbol',
    **period**: '240',
    **range**: 50,
    indicators: [
      { name: 'ACCU_DISTRIB', id: 'STD;Accumulation_Distribution', version: '25' },
  }, (periods) => {
  
  });
});

**Comment by Mathieu2301** [comment, marked as answer]
The 'period' property defines the timeframe, it should takes values such as '1', '3', '5', '15', '30', '45', '60', '120', '180', '240', '1D', '1W' or '1M'.
The 'range' property is the number of loaded periods.

You can see all this information if you have IntelliSense. But I will soon make a complete documentation.

**Reply by shankarcse2010** [reply]
Thanks@Mathieu2301
